NuGet.config

```diff
@@ -9,5 +9,8 @@
     <add key="aspnet-entityframeworkcore" value="https://dotnetfeed.blob.core.windows.net/aspnet-entityframeworkcore/index.json" />
     <add key="aspnet-aspnetcore-tooling" value="https://dotnetfeed.blob.core.windows.net/aspnet-aspnetcore-tooling/index.json" />
     <add key="grpc-nuget-dev" value="https://grpc.jfrog.io/grpc/api/nuget/v3/grpc-nuget-dev" />
+    <add key="roslyn-tools" value="https://dotnet.myget.org/F/roslyn-tools/api/v3/index.json" />
+    <add key="aspnetcore-tools" value="https://dotnet.myget.org/F/aspnetcore-tools/api/v3/index.json" />
+    <add key="aspnetcore-dev" value="https://dotnet.myget.org/F/aspnetcore-dev/api/v3/index.json" />
```

Comment:
@mmitche why were these deprecated feeds added? Nobody has used dotnet.myget.org for a while.

Reply:
Oops I meant @JunTaoLuo and my question is specifically about aspnetcore-dev.

Reply:
Issue here was just that the corresponding `$(RestoreSources)` change hadn't yet been removed